lib/src/main/java/com/diffplug/spotless/SerializableFileFilter.java

```diff
@@ -20,8 +20,8 @@
 
 /** A file filter with full support for serialization. */
 public interface SerializableFileFilter extends FileFilter, Serializable, NoLambda {
-	/** Creates a FileFilter which will accept all files except files with the given name. */
-	public static SerializableFileFilter skipFilesNamed(String name) {
-		return new SerializableFileFilterImpl.SkipFilesNamed(name);
+	/** Creates a FileFilter which will accept all files except files with the given name(s). */
+	public static SerializableFileFilter skipFilesNamed(String... names) {
```

Comment:
It's definitely source-compatible, if not binary.  Good enough while we've got zero users besides ourselves.  Good catch though 👍